README.rst

```diff
@@ -13,411 +13,10 @@ BigQuery DataFrames is an open-source package. You can run
-
-To use BigQuery DataFrames remote functions, you must be granted the
-following IAM roles in the project:
-
-* BigQuery Data Editor (roles/bigquery.dataEditor)
-* BigQuery Connection Admin (roles/bigquery.connectionAdmin)
-* Cloud Functions Developer (roles/cloudfunctions.developer)
-* Service Account User (roles/iam.serviceAccountUser)
-* Storage Object Viewer (roles/storage.objectViewer)
-* Project IAM Admin (roles/resourcemanager.projectIamAdmin) if using default
-  BigQuery connection, or Browser (roles/browser) if using a pre-configured connection.
-  This requirement can be avoided by setting
-  ``bigframes.pandas.options.bigquery.skip_bq_connection_check`` option to ``True``,
-  in which case the connection (default or pre-configured) would be
-  used as-is without any existence or permission check.
-
-**Limitations**
-
-* Remote functions take about 90 seconds to become available when you first create them.
-* Trivial changes in the notebook, such as inserting a new cell or renaming a variable,
-  might cause the remote function to be re-created, even if these changes are unrelated
-  to the remote function code.
-* BigQuery DataFrames does not differentiate any personal data you include in the remote
-  function code. The remote function code is serialized as an opaque box to deploy it as a
-  Cloud Functions function.
-* The Cloud Functions (2nd gen) functions, BigQuery connections, and BigQuery remote
-  functions created by BigQuery DataFrames persist in Google Cloud. If you don’t want to
-  keep these resources, you must delete them separately using an appropriate Cloud Functions
-  or BigQuery interface.
-* A project can have up to 1000 Cloud Functions (2nd gen) functions at a time. See Cloud
-  Functions quotas for all the limits.
-
-
-Quotas and limits
-------------------
-
-`BigQuery quotas <https://cloud.google.com/bigquery/quotas>`_
-including hardware, software, and network components.
-
-
-Session termination
--------------------
-
-Each BigQuery DataFrames DataFrame or Series object is tied to a BigQuery
-DataFrames session, which is in turn based on a BigQuery session. BigQuery
-sessions
-`auto-terminate <https://cloud.google.com/bigquery/docs/sessions-terminating#auto-terminate_a_session>`_
-; when this happens, you can’t use previously
-created DataFrame or Series objects and must re-create them using a new
-BigQuery DataFrames session. You can do this by running
-``bigframes.pandas.close_session()`` and then re-running the BigQuery
-DataFrames expressions.
-
-
-Data processing location
-------------------------
-
-BigQuery DataFrames is designed for scale, which it achieves by keeping data
-and processing on the BigQuery service. However, you can bring data into the
-memory of your client machine by calling ``.to_pandas()`` on a DataFrame or Series
-object. If you choose to do this, the memory limitation of your client machine
-applies.
-
-
-License
```

Comment:
Please keep this information. It's necessary to know that we have third-party code when folks download our package.

Reply:
Done.